packages/p2p-v2/src/components/OnlineStatus/OnlineStatusLabel.tsx

```diff
@@ -3,7 +3,7 @@ import { getLastOnlineLabel } from '@/utils';
 import { Text } from '@deriv-com/ui';
 
 type TOnlineStatusLabelProps = {
-    isOnline: 0 | 1;
+    isOnline: boolean;
```

Comment:
lets make it by default false so user's just need to pass isOnline as a prop and not do `isOnline={false}`

```suggestion
    isOnline?: boolean;
```